source/math/FloatPoint3D.ooc

```diff
@@ -67,7 +67,7 @@ FloatPoint3D: cover {
 	operator <= (other: This) -> Bool { this x <= other x && this y <= other y && this z <= other z }
 	operator >= (other: This) -> Bool { this x >= other x && this y >= other y && this z >= other z }
 	operator as -> String { this toString() }
-	toString: func -> String { "#{this x toString()}, #{this y toString()}, #{this z toString()}" }
+	toString: func -> String { "#{"%.8f" format(this x)}, #{"%.8f" format(this y)}, #{"%.8f" format(this z)}" }
```

Comment:
Same here, use `formatFloat` instead of `format` and the leaks will be all gone.